### DISCUSSION TITLE
BUG: Scrolling on a page jumps around chaotically

### DISCUSSION THREAD

**Discussion by davidhughhenrymack** [GitbookIO/community#302] [0. General bugs and questions]
I'm editing a page in our gitbook

When I've loaded the page, and scroll quickly, the page location keeps jumping. I suspect it might be from lazy loading images.

This is made worse by the page crashing and needing reloaded every few hours.

**Comment by davidhughhenrymack** [comment]
Here's the crash I get from simply having the page open (and I think switching tabs back to it):
<img width="645" alt="Screenshot 2023-08-14 at 1 27 03 PM" src="https://github.com/GitbookIO/community/assets/15009058/0abdcd64-18eb-4542-ac38-9f65b861beea">

Safari, OSX, new Mac Book Pro

**Reply by aikoii** [reply]
Hi @davidhughhenrymack, I'm sorry you are experiencing this! Could you point me to the page where you see this issue? 🙏  If it's easier, feel free to shoot us an email at support@gitbook.com with the details.
